frida_mode/GNUmakefile

```diff
@@ -31,7 +39,9 @@ endif
 
 ifeq "$(shell uname)" "Darwin"
  OS:=macos
- CFLAGS:=$(CFLAGS) -Wno-deprecated-declarations
+ RT_CFLAGS:=$(RT_CFLAGS) -Wno-deprecated-declarations
+else
+ RT_CFLAGS:=$(RT_CFLAGS) -Wno-prio-ctor-dtor
```

Comment:
this needs to be removed. some compilers do not now prio-ctor-dtor

Reply:
Sure no problems. It may lead to warnings in the build though. I assume that’s preferable?